pkg/scheduler/backend/queue/scheduling_queue.go

```diff
@@ -884,7 +884,7 @@ func (p *PriorityQueue) Update(logger klog.Logger, oldPod, newPod *v1.Pod) {
 		// this update may make the Pod schedulable in case it gets rejected and comes back to the queue.
 		// We can clean it up once we change updatePodInSchedulingQueue to call MoveAllToActiveOrBackoffQueue.
 		// See https://github.com/kubernetes/kubernetes/pull/125578#discussion_r1648338033 for more context.
-		if exists := p.activeQ.addEventIfPodInFlight(oldPod, newPod, framework.UnscheduledPodUpdate); exists {
+		if exists := p.activeQ.addEventIfPodInFlight(oldPod, newPod, framework.PodItselfUpdate); exists {
```

Comment:
I know it's an original problem, but we forgot using `framework.PodSchedulingPropertiesChange(newPod, oldPod)` here too.